deployer/src/lib.rs

```diff
@@ -452,6 +453,21 @@ impl StartOptions {
             0
         }
     }
+
+    /// Get the io-engine api versions.
+    pub fn io_api_versions(&self) -> &Vec<IoEngineApiVersion> {
+        &self.io_engine_api_versions
+    }
+    /// Get the latest io-engine api version.
+    pub fn latest_io_api_version(&self) -> IoEngineApiVersion {
+        let mut sorted = self.io_engine_api_versions.clone();
+        sorted.sort();
```

Comment:
Wouldn't this return the oldest api version?
Eg: [v0, v2, v1] -> [v0, v1, v2] and storing the first element would give `v0` whereas the latest should have been `v2`?

Reply:
Yes we need latest one instead.

Reply:
Done.